components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -308,12 +312,37 @@ private void readChildElements(OMElement serverConfig,
                 setThrottleProperties(serverConfig);
             } else if (APIConstants.WorkflowConfigConstants.WORKFLOW.equals(localName)){
                 setWorkflowProperties(serverConfig);
+            } else if (APIConstants.ApplicationAttributes.APPLICATION_ATTRIBUTES.equals(localName)){
+                Iterator iterator = element.getChildrenWithLocalName(APIConstants.ApplicationAttributes.ATTRIBUTE);
+                while (iterator.hasNext()) {
```

Comment:
possibility to occur NPE. so add a check to attribute,  omElement.getChildElements();

Reply:
I checked the implementation of getChildElements(). Iterator does not get null value.